plugins/woocommerce/tests/php/includes/wc-order-functions-test.php

```diff
@@ -110,4 +110,70 @@ public function test_wc_update_total_sales_counts() {
 		$this->assertEquals( 0, wc_get_product( $product_id )->get_total_sales() );
 	}
 
+
+	/**
+	 * Test wc_update_coupon_usage_counts and check usage_count after order reflection.
+	 *
+	 * Tests the fix for issue #31245
+	 */
+	public function test_wc_update_coupon_usage_counts() {
+
+		$coupon = WC_Helper_Coupon::create_coupon( 'test' );
```

Comment:
```suggestion
	public function test_wc_update_coupon_usage_counts() {
		$coupon = WC_Helper_Coupon::create_coupon( 'test' );
```

Appease the code style linter.